<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTeam</class>
 <widget class="QWidget" name="addTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Team</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cityInput"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
